assets/queries/ansible/aws/redshift_not_encrypted/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "Redshift Not Encrypted",
   "severity": "HIGH",
   "category": "Encryption",
-  "descriptionText": "Check if 'encrypted' field is false or undefined (default is false)",
+  "descriptionText": "AWS Redshift Cluster should be encrypted. Check if 'encrypted' field is false or undefined (default is false)",
```

Comment:
Can you also update for `kics\assets\queries\cloudFormation\aws\redshift_not_encrypted\metadata.json`, please?